src/macros.rs

```diff
@@ -102,6 +102,14 @@ macro_rules! mk_impl_lang {
                     )*
                 }
             }
+
+            pub(crate) fn get_ts_language(&self) -> Language {
```

Comment:
Could you add docs like for the function above?

Reply:
With "docs", are you referring to documentation or comments that explain this function?

Reply:
Just comments, like the function just above this one